src/pymovements/dataset/dataset_download.py

```diff
@@ -77,6 +81,15 @@ def download_dataset(
     RuntimeError
         If downloading a resource failed for all given mirrors.
     """
+    disclaimer_text = f"""\
+You are downloading the {definition.long_name}. Please be aware that pymovements does not
```

Comment:
What do you think about adding the short name in parentheses?

```
You are downloading the {definition.long_name} ({definition.name}).
```

Reply:
On second thought, that won't work well for some datasets: 

```
Code Comprehension dataset (Code Comprehension)
```

Although this still would be nice:

```
Beijing Sentence Corpus (BSC)
```

So let's leave it as it is for now, maybe we'll have some further ideas in the future.